eos-add-flatpak-apps-repos

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -e
+# Copyright (C) 2017 Endless Mobile, Inc.
+# Licensed under the GPLv2
+
+# Add the stable Endless apps flatpak repositories after an OS upgrade
+
+TMPDIR=`mktemp -d`
+trap '[ -n "$TMPDIR" ] && rm -rf "$TMPDIR"' EXIT
+
+# Sanity check
+if [ -z ${TMPDIR} ] || [ ! -d ${TMPDIR} ] ; then
+    echo "Error creating temporary directory, exiting"
+    exit 1
+fi
+
+# This information for setting up this repo is stable, so we
+# can just hardcode here the contents of the .flatpakrepo file
+# so that we don't need to rely on a network connection.
+
+# Endless apps runtime Flatpak repository
+cat <<EOF > ${TMPDIR}/eos-sdk.flatpakrepo
+[Flatpak Repo]
+Title=Endless Apps Runtimes
+Url=FIXME https://ostree.endlessm.com/ostree/eos-sdk
+Homepage=http://endlessm.github.io/eos-knowledge-lib/
+Comment=Flatpak runtime for developing offline content apps for Endless
```

Comment:
I'd say that Endless apps should inherently express our values and respect the user with offline support, async internet and all that good stuff. So I'd just say it's a: "Flatpak runtime for developing apps for Endless OS."